x-pack/test/functional/services/ml/anomaly_explorer.ts

```diff
@@ -66,5 +67,34 @@ export function MachineLearningAnomalyExplorerProvider({ getService }: FtrProvid
     async assertSwimlaneViewByExists() {
       await testSubjects.existOrFail('mlAnomalyExplorerSwimlaneViewBy');
     },
+
+    async openAddToDashboardControl() {
+      await testSubjects.click('mlAnomalyTimelinePanelMenu');
+      await testSubjects.click('mlAnomalyTimelinePanelAddToDashboardButton');
+      await testSubjects.existOrFail('mlAddToDashboardModal');
+    },
+
+    async addAndEditSwimlaneInDashboard(dashboardTitle: string) {
+      await this.filterWithSearchString(dashboardTitle);
+      await testSubjects.isDisplayed('mlDashboardSelectionTable > checkboxSelectAll');
+      await testSubjects.click('mlDashboardSelectionTable > checkboxSelectAll');
+      await testSubjects.isChecked('mlDashboardSelectionTable > checkboxSelectAll');
+      await testSubjects.clickWhenNotDisabled('mlAddAndEditDashboardButton');
+      const embeddable = await testSubjects.find('mlAnomalySwimlaneEmbeddableWrapper');
+      const swimlane = await embeddable.findByClassName('ml-swimlanes');
+      expect(await swimlane.isDisplayed()).to.be(true);
```

Comment:
I'd suggest to change this to `to.eql(true, 'Anomaly swimlane should be displayed in dashboard')`, so we have a better understandable error output (instead of `expected 'true'  but got ' false').